docs/getting-started/pre-requisite/gosdk.md

```diff
@@ -111,8 +111,8 @@ type BCDB interface {
 
 ### Source Code
 
-Now, that we can created a connection and received the `BCDB` implementation, we can open a database session by calling the `Session()` method.
-The following function opens a database session for a given connection.
+Now, once we created the Orion connection and received the `BCDB` object instance, we can open a database session by calling the `Session()` method. The `Session` object authenticate database user again database server. 
```

Comment:
The `Session` object authenticates the database user against the database server.